profiles:
  ompl_default:
    pipeline_id: "ompl"
    planner_id: "ompl"
    allowed_planning_time_sec: 5.0
    num_planning_attempts: 10
    executor_mode: "moveit"

  trajopt_default:
    pipeline_id: "rlc_trajopt"
    planner_id: "rlc_trajopt"
    allowed_planning_time_sec: 20.0
    num_planning_attempts: 10
    executor_mode: "moveit"